scheduler/scheduler.go

```diff
@@ -517,7 +517,17 @@ func (s *Scheduler) splitTasksByDistro(tasksToSplit []task.Task) (
 
 		if t.DistroId == "" {
 			// this is a lazy way to backfill distro names on tasks.
-			t.SetDistro(distrosToUse[0])
+			if err = t.SetDistro(distrosToUse[0]); err != nil {
+				grip.Info(message.WrapError(err, message.Fields{
```

Comment:
`grip.Error`, I think.

Reply:
discussed: okay since it's continue on error